phpunit.xml.dist

```diff
@@ -34,29 +35,28 @@
         <env name="BITBUCKET_AUTH_METHOD" value="http_password"/>
         <env name="BITBUCKET_TOKEN" value=""/>
         <env name="BITBUCKET_SECRET" value=""/>
-
-        <!-- ###+ symfony/framework-bundle ### -->
-        <env name="APP_ENV" value="test"/>
-        <env name="APP_SECRET" value="6d878195836abc1bd87b92206d7db9d8"/>
-        <env name="SYMFONY_DEPRECATIONS_HELPER" value="max[self]=0" />
-        <!-- env name="TRUSTED_PROXIES" value="127.0.0.1,127.0.0.2" -->
-        <!-- env name="TRUSTED_HOSTS" value="'^localhost|example\.com$'" -->
-        <!-- ###- symfony/framework-bundle ### -->
     </php>
 
     <testsuites>
         <testsuite name="Project Test Suite">
-            <directory>tests/</directory>
+            <directory>tests</directory>
         </testsuite>
     </testsuites>
 
-    <coverage>
-        <include>
-            <directory suffix=".php">./src/</directory>
-        </include>
-    </coverage>
+    <filter>
```

Comment:
this is an old (and deprecated) configuration value. We need to kee "coverage", "include", etc.

Reply:
What? Are you sure? Now we use phpunit 8.5 and symfony/phpunit-bridge v5.2.6 with his recipe.

Reply:
Well, we should use phpunit 9.5 instead

Reply:
Ok! We can do it in different PR?

Reply:
So you're proposing to change this configuration here and then revert such change in a future PR?
Maybe it's better not to change it at all...

Reply:
I'm saying that now everything is green and finally the tests are also running locally... before no!

Reply:
Do you think that it depends on phpunit version? Is there a special reason to stay on 8.5 instead of latest 9.5?

Reply:
Reverted! ;)

Reply:
With phpunit 9.5 `DG\BypassFinals::enable();` not work! :(

Reply:
Weird, I'm using it on many projects